<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationCollection</class>
 <widget class="QMainWindow" name="ApplicationCollection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application Collection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/Main Window Icon.png</normaloff>Icons/Main Window Icon.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>-430</x>
      <y>750</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColTransverseDesigner">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Column Transverse 
Designer</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
   <property name="statusTip">
    <string>Testing Lang</string>
   </property>
  </action>
  <action name="actionTemp1">
   <property name="text">
    <string>Designer1</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
